sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -629,6 +638,7 @@ class ServiceBusReceivedMessage(ServiceBusMessage):
 
     def __init__(self, message, receive_mode=ServiceBusReceiveMode.PEEK_LOCK, **kwargs):
         # type: (uamqp.message.Message, Union[ServiceBusReceiveMode, str], Any) -> None
+        message = create_messages_from_dicts_if_needed(message, ServiceBusMessage)
```

Comment:
I don't think we need the conversion here as users are not supposed to instantiate `ServiceBusReceivedMessage` by them selves.